Username-Password-Flow/src/main/java/TokenCacheAspect.java

```diff
@@ -4,18 +4,12 @@
 import com.microsoft.aad.msal4j.ITokenCacheAccessAspect;
 import com.microsoft.aad.msal4j.ITokenCacheAccessContext;
 
-import java.net.URI;
-import java.net.URL;
-import java.nio.file.*;
-import java.util.HashMap;
-import java.util.Map;
-
 public class TokenCacheAspect implements ITokenCacheAccessAspect {
```

Comment:
if we want to use in-memory cache we do not need TokenCacheAspect

Reply:
I got rid of TokenCacheAspect and any hardcoded accounts in the latest commit.